docs/dptp-triage-sop/infrastructure-jobs.md

```diff
@@ -237,7 +238,42 @@ When the job fails, it will list all repos that are inaccessible by the bots at
 
 #### Resolution
 
-Reach out to the owner(s) for each 
+Reach out to the owner(s) for each
 repo listed asking them to grant the bots permissions by following the docs.
 
-If there is argument that they do not want the bots to be added to a repo or org, it can be passed to the job as an additional `ignore` parameter.
+If there is argument that they do not want the bots to be added to a repo or org, it can be passed to the job as an
+additional `ignore` parameter.
+
+## `periodic-openshift-release-private-org-sync`
+
+This job runs [`private-org-sync`](https://github.com/openshift/ci-tools/tree/master/cmd/private-org-sync) to
+sync `openshift-priv`
+mirror repos with their respective "public" repos.
+
+#### Useful Links
+
+- [Recent executions on Deck-Internal](https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/?job=periodic-openshift-release-private-org-sync)
+- [infra-periodics.yaml (ProwJob configuration)](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml)
+
+### Failed to push to destination
+
+#### Symptom
+
+```
+level=error msg="failed to push to destination, no retry possible" branch=release-4.14 destination=openshift-priv/file-integrity-operator@release-4.14
+error="" exit-code=1 local-repo=/tmp/1719126826/openshift/file-integrity-operator org=openshift output="To https://github.com/openshift-priv/file-integrity-operator\n
+6e4fa791..c5b715ed  FETCH_HEAD -> release-4.14\n ! [rejected]          
+v1.2.0 -> v1.2.0 (already exists)\nerror: failed to push some refs to 'https://github.com/openshift-priv/file-integrity-operator'\n
+hint: Updates were rejected because the tag already exists in the remote.\n"
+```
+
+#### Culprit
+
+The private mirror repo already contains a tag that the tool is attempting to mirror to it from the public repo.
```

Comment:
Btw, the org-sync tool doesn't mess up with tags, does it?

Reply:
Evidently it mirrors the tags. This is the first time I have looked at this tool.